### ISSUE TITLE
oauth2 authentication ignores client/secret 

### ISSUE THREAD

**Issue by DavidBiesack** [sassoftware/unravl#89]
If you use the clientId and clientSecret in an 

```
    "auth" : {"oath2" : host, ...  "user" : "userid-string", "password" : "password-string"} 
```

object, an embedded clientId/clientSecret pair is ignored, or if clientid/secret exist in the .netrc file, they are not read.

**Comment by DavidBiesack** [comment]
@sweeneyb see the PR #91 for a fix for this issue

**Comment by DavidBiesack** [comment]
Fixed by PR #91
